bin/generate_valid_config_file.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+"""Generates a validation configuration file from JSON struct data"""
+
+# Note:  This is written in Python instead of Julia because the Julia YAML
+# package does not support writing (only reading).
+
+import json
+import os
+import sys
+import pdb
+from collections import OrderedDict
+
+
+import yaml
+
+
+POWER_SYSTEM_DESCRIPTOR_FILE = os.path.join(
+    "src",
+    "descriptors",
+    "power_system_structs.json"
+)
+
+def setup_yaml():
+  """ https://stackoverflow.com/a/8661021 """
+  represent_dict_order = lambda self, data:  self.represent_mapping('tag:yaml.org,2002:map', data.items())
+  yaml.add_representer(OrderedDict, represent_dict_order)
+setup_yaml()
+
+def read_json_data(filename):
+    """Return the JSON data from a file."""
+    with open(filename) as fp_in:
+        return json.load(fp_in)
+
+
+def generate_config(input_file):
+    """Generate validation descriptors from the PowerSystems struct data file."""
+    config = {}
+    data = read_json_data(input_file)
+    items=[]
```

Comment:
nit: spaces around '='

On the next line it looks like you forgot to delete 'structName'